echopype/convert/set_groups_ek60.py

```diff
@@ -321,17 +321,18 @@ def set_platform(self, NMEA_only=False) -> xr.Dataset:
                             self.parser_obj.ping_time[ch],
                             {
                                 "axis": "T",
-                                "long_name": "Timestamps for position datagrams",
+                                "long_name": "Timestamps for platform motion and orientation data",
                                 "standard_name": "time",
-                                "comment": "Time coordinate corresponding to platform sensors.",
+                                "comment": "Time coordinate corresponding to platform motion and "
+                                "orientation sensors.",
```

Comment:
I suggest the following to explicitly state the source of data
```suggestion
                                "comment": "Time coordinate corresponding to platform motion and "
                                "orientation data. For EK60, this data is saved in the RAW0 sample datagrams "
                                "along with the backscatter data",
```

Reply:
I think the second sentence you add may not be appropriate. If we also use `set_group_ek60.py` for ES70, then I think that comment really does not apply? For the other comments where you specifically mention the sensor, I fell the same way.

Reply:
ES70 and the other sonar models do use the same set of datagram specifiers. For example ES70 uses RAW0 as well.

But this is fine. Let's go withouts those. You can merge this after you have everything you feel comfortable with in it.

It's not difficult to code those in, but since it does take changing other part of the code to get the right source for other sonar models, let's do it post v0.6.0.